helm-charts/FATE/values-template-example.yaml

```diff
@@ -29,15 +30,22 @@ backend: eggroll
 # rollsite: 
   # type: NodePort
   # nodePort: 30091
-  # exchange:
-    # ip: 192.168.0.1
-    # port: 30000
```

Comment:
The new roll site is lbrollsite, so the original rollsite does not need to be modified

Reply:
Fixed